packages/node-core/src/indexer/dictionary/v1/dictionaryV1.ts

```diff
@@ -0,0 +1,262 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import {ApolloClient, ApolloLink, gql, HttpLink, InMemoryCache, NormalizedCacheObject} from '@apollo/client/core';
+import {EventEmitter2} from '@nestjs/event-emitter';
+import {dictHttpLink} from '@subql/apollo-links';
+import {DictionaryQueryCondition, DictionaryQueryEntry as DictionaryV1QueryEntry} from '@subql/types-core';
+import {buildQuery, GqlNode, GqlQuery, GqlVar, MetaData as DictionaryV1Metadata} from '@subql/utils';
+import fetch from 'cross-fetch';
+import {NodeConfig} from '../../../configure';
+import {IndexerEvent} from '../../../events';
+import {getLogger} from '../../../logger';
+import {profiler} from '../../../profiler';
+import {timeout} from '../../../utils';
+import {CoreDictionary} from '../coreDictionary';
+import {DictionaryResponse} from '../types';
+import {buildDictQueryFragment, distinctErrorEscaped, startHeightEscaped} from './utils';
+
+const logger = getLogger('dictionary v1');
```

Comment:
To make it easier to use the debug flag we should remove spaces. There are a couple of other occurrences i think
```suggestion
const logger = getLogger('dictionary-v1');
```